src/sentry/migrations/0028_user_reports.py

```diff
@@ -18,9 +27,21 @@ def backfill_user_reports(apps, schema_editor):
     user_reports = UserReport.objects.filter(group__isnull=True, environment__isnull=True)
 
     for report in RangeQuerySetWrapper(user_reports, step=1000):
-        event = eventstore.get_event_by_id(report.project_id, report.event_id)
+        try:
+            event = eventstore.get_event_by_id(report.project_id, report.event_id)
+        except (SnubaError, QueryOutsideGroupActivityError, QueryOutsideRetentionError) as se:
+            logger.warn(
+                "failed to fetch event {} for project {}: {}".format(
```

Comment:
Needs to be unicode. Otherwise this itself could raise an exception. Or don't use `str.format`.